python/py-ds-store/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
```

Comment:
is this not available on PyPI - that is the preferred source unless there is a reason that doesn't work.

Reply:
No, it is. I think I just needed to set the distname appropriately.